src/pages/Homepage/LendByCategoryHomepage.vue

```diff
@@ -335,11 +337,27 @@ export default {
 <style lang="scss" scoped>
 @import 'settings';
 
+// utils
+.section {
+	position: relative;
+	padding: 3rem 0;
+
+	@include breakpoint(large) {
+		padding: 6rem 0;
+	}
+}
+
 .lend-by-category-homepage {
 	overflow: hidden;
 }
 
 .featured-loans {
+	padding: 2rem 0 1rem;
```

Comment:
This section and `.loan-categories` dont have the same padding as per the design